code/Pagetypes/FreshdeskPage.php

```diff
@@ -0,0 +1,262 @@
+<?php
+
+
+class FreshdeskPage extends Page
+{
+}
+
+class FreshdeskPage_Controller extends Page_Controller
+{
+    /**
+     * Status mapping for Freshdesk tickets.
+     *
+     * @var array
+     * @config
+     */
+    private static $freshdeskStatus = [
```

Comment:
There seems to be a confusion here - on one hand we have FreshdeskTicketStatuses that roll out a proper model for these things. On the other hand we have some things hardcoded here. Are these things the same thing?

Also, these number are part of Freshdesk, so they should be rolled into Freshdesk API, Controller shouldn't need to know.

Reply:
ack